react/react1/week1/todo_list/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+function TodoItem(props) {
+  return (
+    <div>
+      <h3>{props.description}</h3>
+      <p>{props.deadline}</p>
+    </div>
+  );
+}
+
+function App() {
+  const todoList = [
+    {
+      description: 'Get out of bed',
+      deadline: 'Wed Sep 13 2017'
+    },
+    {
+      description: 'Brush teeth',
+      deadline: 'Thu Sep 14 2017'
+    },
+    {
+      description: 'Eat breakfast',
+      deadline: 'Fri Sep 15 2017'
+    }
+  ];
```

Comment:
You should probably want to move it to a separate file

Reply:
Ok, I add new file, the name is TodoList.js and move these codes there. Thanks